IdentityCore/src/util/MSIDKeyOperationUtil.m

```diff
@@ -0,0 +1,129 @@
+//
+//  MSIDKeyOperationUtil.m
+//  IdentityCore
+//
+//  Created by Ameya Patil on 9/24/21.
+//  Copyright © 2021 Microsoft. All rights reserved.
+//
+
+#import "MSIDKeyOperationUtil.h"
+#import "MSIDJwtAlgorithm.h"
+#import "NSData+JWT.h"
+#import "NSData+MSIDExtensions.h"
+
+@implementation MSIDKeyOperationUtil
+
++ (MSIDKeyOperationUtil *)sharedInstance
+{
+    static dispatch_once_t once;
+    static MSIDKeyOperationUtil *s_keyOperationUtil = nil;
+    dispatch_once(&once, ^{
+        s_keyOperationUtil = [[MSIDKeyOperationUtil alloc] init];
+    });
+    return s_keyOperationUtil;
+}
+
+- (BOOL)isKeyFromSecureEnclave:(SecKeyRef)key
+{
+    if (key)
+    {
+        NSDictionary *attributes = (NSDictionary *)CFBridgingRelease(SecKeyCopyAttributes(key));
+        NSString *attrTokenId = (NSString *)[attributes valueForKey:(NSString *)CFBridgingRelease(kSecAttrTokenID)];
```

Comment:
nit, I think (__bridge id) is better after valuesForKey.

Reply:
Thanks, can you explain why it is better for my knowledge? I know, at least in this case, that it will be a string called 'com.apple.settoken' for key from secure enclave and nil otherwise.